tutorials/deploy-remote-desktop-ubuntu-2204/index.mdx

```diff
@@ -0,0 +1,175 @@
+
+  ```
+  apt install xrdp ufw
+  ```
+
+5. The package manager configures the service to be automatically started upon system reboot. Run the following command to check that the service has been started during the installation:
+
+  ```
+  systemctl status xrdp
+  ```
+
+  A message like the following displays, confirming the status `active (running)`:
+
+  ```
+  ● xrdp.service - xrdp daemon
+       Loaded: loaded (/lib/systemd/system/xrdp.service; enabled; vendor preset: enabled)
+       Active: active (running) since Fri 2022-05-06 14:40:08 UTC; 47s ago
+  ```
+
+  In case the service has not been started automatically, start it with `systemctl`:
+
+  ```
+  systemctl start xrdp
+  ```
+
+6. Open the xRDP configuration file in a text editor of your choice, for example `nano`:
+
+  ```
+  nano /etc/xrdp/xrdp.ini
+  ```
+
+5. Scroll to the bottom of the file and add the following line, then save the file and exit the text editor:
+
+  ```
+  exec startxfce4
+  ```
+
+6. Restart the xrdp service using `systemctl` to activate the new configuration:
+
+  ```
+  systemctl restart xrdp
+  ```
+
+## Configuring the firewally
+
+1. Restrict the access to port `3389` to an IP address or a range of IPs. To allow access from the block `192.168.1.0/24` launch the following command:
+
+  ```
+  ufw allow from 192.168.1.0/24 to any port 3389
+  ```
+
+  To allow connections from any host run the following command:
+
+  ```
+  ufw allow 3389
+  ```
+  <Message type="important">
+
+  Exposing the service to any IP may cause a security issue.
+
+  </Message>
+
+2. Enable the firewall:
+
+  ```
+  ufw enable
+  ```
+
+  <Message type="tip">
+
+  For more information on how to configure the firewall of your Instance, you can [read our tutorial about the configuration of ufw on Ubuntu](/tutorials/installation-uncomplicated-firewall/).
+
+  </Message>
+
+## Configuring a regular user
+
+It is recommended to create a regular user to connect to the remote desktop, to avoid security issues.
+
+1. Add the user with the `adduser` command:
+
+  ```
+  adduser <myuser>
+  ```
+
+2. (Optionally) If you want to give superuser rights to the user, install sudo using the `apt` package manager:
+
+  ```
+  apt install sudo
+  ```
+
+3. (Optionally) Add the user to the sudoers file:
+
+  ```
+  adduser <myuser> sudo
+  ```
+
+## Connecting to the remote desktop
+
+1. Download and install a RDP client of your choice. We will use the [Microsoft Remote Desktop Client](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-clients) in this tutorial.
+
+2. Start the RDP client and click **Add Desktop** to add a new connection :
+
+  <Lightbox src="scaleway-rdp_01.webp" alt="" />
+
+3. Enter the connection details. You need to enter at least your Instances IP in the **PC name**. Optionally you can add a **Friendly Name** to facilitate the identification of your instance in case you configure several RDP connections.
```

Comment:
```suggestion
3. Enter the connection details. You must enter at least your Instance's IP under **PC name**. Optionally, you can add a **Friendly Name** to facilitate the identification of your instance in case you configure several RDP connections.
```